content/author/mr-imint.md

```diff
@@ -0,0 +1,6 @@
+---
+date: 2017-09-12T15:40:00+02:00
+title: Mr IMINT
+---
+
+_Mr IMINT_ is a small shareholder of Imint AB. He lives in Oslo, Norway, and have no current or prior connection to Imint AB.
```

Comment:
It should be: "and  _has_ no current"

Reply:
As you spend quite some time on doing Imint related research I think there you can mention your areas of interest when it comes to Imint.

Reply:
Ok, I'll fix the errors, and update it a bit.